### PR TITLE
Added link to stator

### PR DESCRIPTION
**Pull request by yann510**
Stator solves all the commonly found problems of creating a new full-stack project: `configurations`, `coding guidelines`, `releases`, `deployment` and more...

Extract from the project's readme:

## 📚 About the Project

Have you ever started a new project by yourself?
If so, you probably know that it is tedious to set up all the necessary tools.
Just like you, the part I enjoy the most is coding, not boilerplate.

stator solves all of this for you by using the **latest technologies' most up-to-date standards**.
Indeed, this template is opinionated as to what's the best, but it does enforce excellent practices such as **code re-usability**, **enforces coding guidelines**, **usage of a monorepo**, etc.

If you want more details about how this idea was implemented, I recommend reading the [series of blog articles](https://yann510.hashnode.dev/creating-the-modern-developer-stack-template-part-1-ckfl56axy02e85ds18pa26a6z) I wrote on the topic.

### CODE REVIEW

**Comment by GabrielBottari** [comment]
Our project is growing quick! Already 50 stars in less than a month :)

File: README.md [hunk 1]
```diff
@@ -98,6 +98,7 @@
   - [NestJS GraphQL Boilerplate](https://github.com/fernandohenriques/nestjs-graphql-boilerplate) - Dockerized API boilerplate with NestJS, TypeORM, TypeGrapQL, MongoDB, GraphQL and automated tasks with Makefile. Code first approach.
   - [The Knests Stack](https://github.com/tudorconstantin/knests/) - Full stack/end starter with: PostgreSQL, Knex.js, NestJS, Next.js, GraphQL, React, Material-UI, Docker multistage images for, Docker compose and a GitLab CI/CD pipeline fully configured.
   - [Nest Hackathon Starter](https://github.com/ahmetuysal/nest-hackathon-starter) - Hackathon starter project for NestJS. Includes TypeORM, email verification, Passport-JWT authentication, Swagger and more.
+  - [Stator](https://github.com/chocolat-chaud-io/stator) - A full-stack boilerplate that does it all - automatic releases, deployments, conventions enforced
```
Comment (juliandavidmr) [inline README.md:101]: Please add ending in `.`

Comment (yann510) [inline README.md:101]:
```suggestion
  - [Stator](https://github.com/chocolat-chaud-io/stator) - A full-stack boilerplate that does it all - automatic releases, deployments, conventions enforced.
```

Comment (yann510) [reply README.md]: Done ;)

**Comment by juliandavidmr** [comment]
Thank you for your contribution.

**Pull request merged**